services/scabbard/src/cli/main.rs

```diff
@@ -748,6 +814,107 @@ fn run() -> Result<(), CliError> {
 
                 Ok(client.submit(circuit, service_id, batch_list, Some(wait))?)
             }
+            ("list", Some(matches)) => {
+                let url = matches.value_of("url").expect("default not set for --url");
+                let client = ScabbardClient::new(url);
+
+                let circuit = matches
+                    .value_of("circuit")
+                    .ok_or_else(|| CliError::MissingArgument("circuit".into()))?;
+                let service_id = matches
+                    .value_of("service-id")
+                    .ok_or_else(|| CliError::MissingArgument("service-id".into()))?;
+
+                let format = matches
+                    .value_of("format")
+                    .expect("default not set for --format");
+
+                let registries = client
+                    .get_state_with_prefix(circuit, service_id, Some("00ec01"))?
```

Comment:
Ultimately, this namespace should be defined as a constant in the Sabre SDK and just pulled in by the CLI.